### PR TITLE
Avoid to consume the click event when user click outside the tooltip

### PR DESCRIPTION
**Pull request by jDramaix**
- Give the ability to the developer to disable a tooltip in the BeforeShowTooltipEvent

### CODE REVIEW

**Comment by jDramaix** [comment]
This is needed for Roler

File: tooltip/src/main/java/com/arcbees/gquery/tooltip/client/TooltipImpl.java [hunk 1]
```diff
@@ -49,11 +52,8 @@
 import com.google.web.bindery.event.shared.HandlerRegistrations;
 import com.google.web.bindery.event.shared.SimpleEventBus;
 
-import static com.arcbees.gquery.tooltip.client.Tooltip.Tooltip;
-import static com.arcbees.gquery.tooltip.client.Tooltip.getImpl;
-import static com.google.gwt.dom.client.BrowserEvents.CLICK;
-import static com.google.gwt.query.client.GQuery.$;
-import static com.google.gwt.query.client.GQuery.document;
+import java.util.ArrayList;
+import java.util.List;
```
Comment (meriouma) [inline tooltip/src/main/java/com/arcbees/gquery/tooltip/client/TooltipImpl.java]: Not sure that's the good import scheme?

**Comment by meriouma** [comment]
LGTM

**Comment by Chris-V** [comment]
lgtm

**Comment by jDramaix** [comment]
@meriouma we should enable checkstyle for this project.

I will create an issue for that

**Comment by meriouma** [comment]
Agreed!

**Pull request merged**